app/src/main/res/values/colors.xml

```diff
@@ -53,7 +61,7 @@
     <color name="sections_gray">#a5a5a5</color>
 
     <color name="color_primary">@color/tum_500</color>
-    <color name="color_primary_dark">@color/tum_800</color>
+    <color name="color_primary_dark">@color/tum_700</color>
```

Comment:
I think `color_primary` and `color_primary_dark` are not actually that blue anymore...

New TUM style is more pure white as primary color, with that gray as dark primary.
The TUM blue is only used as something like an accent color nowadays...

Reply:
I’ll have a look at the style guide to see how we can incorporate it.

Reply:
We’re mostly using the TUM blue as an accent color (e.g. button colors, status bar). It’s not used for any surfaces.